lib/screenplay_web/templates/layout/app.html.heex

```diff
@@ -34,6 +34,8 @@
     <% end %>
     <%= csrf_meta_tag() %>
     <title>Screenplay</title>
+    <link rel="icon" href={Routes.static_path(@conn, "/favicon.ico")} sizes="32x32">
+    <link rel="icon" href={Routes.static_path(@conn, "/images/favicon-#{@environment_name || "prod"}.svg")} type="image/svg+xml">
```

Comment:
I'm not certain if it's just a local bug for me, but my browser can't seem to load the prod favicon when running the branch. Dev and dev-green work though

Reply:
Yeah nevermind, maybe a weird caching thing or something? Works now after directly loading the path to the favicon once